frontend_vue/src/components/ui/DeleteTokenButton.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div class="flex flex-row justify-between items-center mt-32">
+    <div class="flex flex-col">
+      <span>Delete token</span>
+      <span class="text-xs leading-4 text-grey-500 pr-[3rem]">Remove this token and delete all related alerts</span>
+    </div>
+    <div> <base-button variant="danger" @click="openDeleteModal">Delete</base-button></div>
+  </div>
+  <BaseModal
```

Comment:
Can we move this into a separate component?

Reply:
Yeah, We can if you have strong feelings. 

I do not think it was needed though since the modal is fully local to the button

Reply:
uhm but why this is located in the button?
They are two different components, right?

Reply:
I'd go for a ModalDeleteToken component and place it inside the ManageToken page.
The button can trigger the modal by using the useModal hook.
This way you clearly divide the roles, and make the different parts easier to find and eventually edit.

Reply:
Okay. I'll update